unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/PerformanceMeterController/PerformanceMeterController.cs

```diff
@@ -0,0 +1,282 @@
+                + "\n * System Info -> Graphics Device Name -> " + SystemInfo.graphicsDeviceName
+                + "\n * System Info -> System RAM Size -> " + SystemInfo.systemMemorySize
+                + "\n * General Settings -> Auto Quality ON -> " + Settings.i.generalSettings.autoqualityOn
+                + "\n * General Settings -> Scenes Load Radius -> " + Settings.i.generalSettings.scenesLoadRadius
+                + "\n * Quality Settings -> FPSCap -> " + Settings.i.currentQualitySettings.fpsCap
+                + "\n * Quality Settings -> Bloom -> " + Settings.i.currentQualitySettings.bloom
+                + "\n * Quality Settings -> Shadow -> " + Settings.i.currentQualitySettings.shadows
+                + "\n * Quality Settings -> Antialising -> " + Settings.i.currentQualitySettings.antiAliasing
+                + "\n * Quality Settings -> Base Resolution -> " + Settings.i.currentQualitySettings.baseResolution
+                + "\n * Quality Settings -> Color Grading -> " + Settings.i.currentQualitySettings.colorGrading
+                + "\n * Quality Settings -> Display Name -> " + Settings.i.currentQualitySettings.displayName
+                + "\n * Quality Settings -> Render Scale -> " + Settings.i.currentQualitySettings.renderScale
+                + "\n * Quality Settings -> Shadow Distance -> " + Settings.i.currentQualitySettings.shadowDistance
+                + "\n * Quality Settings -> Shadow Resolution -> " + Settings.i.currentQualitySettings.shadowResolution
+                + "\n * Quality Settings -> Soft Shadows -> " + Settings.i.currentQualitySettings.softShadows
+                + "\n * Quality Settings -> SSAO Quality -> " + Settings.i.currentQualitySettings.ssaoQuality
+                + "\n * Quality Settings -> Camera Draw Distance -> " + Settings.i.currentQualitySettings.cameraDrawDistance
+                + "\n * Quality Settings -> Detail Object Culling Enabled -> " + Settings.i.currentQualitySettings.enableDetailObjectCulling
+                + "\n * Quality Settings -> Detail Object Culling Limit -> " + Settings.i.currentQualitySettings.detailObjectCullingLimit
+            );
+
+            // Step 2 - report processed data
+            Log("Data report step 2 - Processed values:"
+                + "\n * PERFORMANCE SCORE (0-10) -> " + CalculatePerformanceScore()
+                + "\n * average FPS -> " + averageFPS
+                + "\n * highest FPS -> " + highestFPS
+                + "\n * lowest FPS -> " + lowestFPS
+                + "\n * 50 percentile (median) FPS -> " + percentile50FPS
+                + "\n * 95 percentile FPS -> " + percentile95FPS
+                + "\n * total hiccups -> " + totalHiccupFrames
+                + "\n * total hiccups time (seconds) -> " + totalHiccupsTimeInSeconds
+                + "\n * total frames -> " + totalFrames
+                + "\n * total frames time (seconds) -> " + totalFramesTimeInSeconds
+            );
```

Comment:
Only as suggestion for future improvements of this tool: Maybe it would be a good idea extract all this info as a template in a text file and define different tags in it (for example: `#OPERATING_SYSTEM`, `#DEVICE_NAME`, `#GRAPHICS_DEVICE`...) in order to be replaced by the `ReportData()` function.

In this way the code implementation would looks cleaner and we would be able to design the performance template in a separated way. Today it would be a simple text file, but maybe in the future we want to create a HTML template that show us the info in a more pretty way :)

Reply:
Good Idea! I'll add a "TODO" comment to tackle this enhancement in the future